config.json

```diff
@@ -186,6 +186,18 @@
       "values": [ "debug", "release", "checked" ],
       "defaultValue": "debug"
     },
+    "EnableProfileGuidedOptimization": {
+      "description": "Enables IBC profile optimizations if profile data are available.",
+      "valueType": "property",
+      "values": [ true, false ],
+      "defaultValue": false
+    },
+    "OptimizationDataDir": {
+      "description": "Sets the path where the build should look for IBC profile data.",
+      "valueType": "property",
+      "values": [],
+      "defaultValue": ""
+    },
```

Comment:
Ah, I see what you mean. I originally tried the `-- /p:Foo=bar` syntax too, but that turned out to be too fragile; there can only be one `--`, and it makes the arguments even more sensitive to ordering. One of the early CI failures was caused by this (it was trying to pass `-rebuild` to `run.exe` on the command line, after I had already fed it `--` followed by msbuild args). I'd much rather keep it in its current form until the shared configs are ready.